src/frame/stylesheets/conversation.less

```diff
@@ -266,24 +267,27 @@
             }
         }
         &.right-row {
-            .msg, .msg-image {
-                background-color: @blue;
-                float: right;
-                color: white;
-                max-width: 204px;
-
-                &:after {
-                    left: 100%;
-                    border-color: rgba(0, 174, 255, 0);
-                    border-left-color: inherit;
-                }
-                a.link, a.link:visited {
+            .msg-wrapper {
+                flex-direction: row-reverse;
```

Comment:
I mean, I understand that it reverses the elements, but why?

Reply:
it aligns the bubble on the right.

Reply:
well.. that's the result of it.. but it just tells the element to start displaying from the right instead of from the left.. if we had appUsers avatars, we could write the exact same markup as for appMakers bubble, but the avatar would be displayed on the right first and everything would be reversed.